prow/cmd/checkconfig/main_test.go

```diff
@@ -2281,6 +2281,49 @@ func TestVerifyLabelPlugin(t *testing.T) {
 			},
 			expectedErrorMsg: "the following orgs or repos have configuration of label plugin using the empty string as label name in restricted labels: orgRepo1, orgRepo2",
 		},
+		{
+			name: "invalid when additional and restricted labels are the same",
+			label: plugins.Label{
+				AdditionalLabels: []string{"cherry-pick-approved"},
+				RestrictedLabels: map[string][]plugins.RestrictedLabel{
+					"orgRepo1": {
+						{
+							Label:        "cherry-pick-approved",
+							AllowedTeams: []string{"some-other-team"},
+						},
+					},
+				},
+			},
+			expectedErrorMsg: "the following orgs or repos have configuration of label plugin using the restricted label cherry-pick-approved which is also configured as an additional label: orgRepo1",
+		},
+		{
+			name: "invalid when additional and restricted labels are the same in multiple orgRepos and empty string",
+			label: plugins.Label{
+				AdditionalLabels: []string{"cherry-pick-approved"},
+				RestrictedLabels: map[string][]plugins.RestrictedLabel{
+					"orgRepo1": {
+						{
+							Label:        "cherry-pick-approved",
+							AllowedTeams: []string{"some-other-team"},
+						},
+					},
+					"orgRepo2": {
+						{
+							Label:        "",
```

Comment:
No need for the empty field here.

Reply:
empty field left here explicitly and mentioned in the test description to show the interaction between two types of errors. Do you want to delete it?

Reply:
exactly the same is marked at line 2270 and 2277

Reply:
It's fine, let's leave it to make the test more readable.